### PR TITLE
@alloy=> Some more minor QA

### PR DESCRIPTION
**Pull request by mzikherman**
Closes https://github.com/artsy/force/issues/759
Closes https://github.com/artsy/force/issues/754
Closes https://github.com/artsy/force/issues/750

Here's the mailcheck hint:

screen shot 2017-01-25 at 1 05 53 pm

### CODE REVIEW

**Comment by alloy** [comment]
Sweet 👍 

Only thing I don’t understand, looking at that screenshot, is the suggestion also doesn’t have a TLD. What gives?

**Comment by mzikherman** [comment]
That's just https://github.com/mailcheck/mailcheck , I am not super familiar with it but it matches what we use on the existing Sign-up/Log-in stuff.

I can investigate it further but I am not sure. This does make the embedded signup consistent with the modal however.

**Comment by mzikherman** [comment]
Actually, let me rephrase my last comment!

What I meant: I agree that the current Mailcheck seems a bit less than ideal (for instance it fixes 'gnail.com', but still lets some invalid ones through). If we want to improve/revamp the behavior of it, I believe that's beyond the scope of this relatively simple QA PR, and can be issued and dealt with separately.

Sorry for the misunderstanding!

**Comment by alloy** [comment]
>  If we want to improve/revamp the behavior of it, I believe that's beyond the scope of this relatively simple QA PR.

Agreed 👍

Filed a new ticket https://github.com/artsy/force/issues/771
